### PR TITLE
Rephrase mailing lists section to avoid incorrect subscribe emails

### PR DESCRIPTION
**Pull request by scholzj**
The user@ and dev@ mailing lists seem to receive lot of emails from users who are not subscribed but are trying to subscribe. I wonder it is has something to do with the UX of the website and if the change in this PR can help to make it more clear to the users that they subscribe on a different email.

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -11,16 +11,16 @@
 		</p>
 		<ul>
 			<li>
-			  <a href="mailto: users@kafka.apache.org">users@kafka.apache.org</a>: A list for general user questions about Kafka&reg;. To subscribe, send an email to <a href="mailto: users-subscribe@kafka.apache.org">users-subscribe@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-users">here</a>.
+				<b>User maling list</b>: A list for general user questions about Kafka&reg;. To subscribe, send an email to <a href="mailto: users-subscribe@kafka.apache.org">users-subscribe@kafka.apache.org</a>. Once subscribed, send your emails to <a href="mailto: users@kafka.apache.org">users@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-users">here</a>.
```
Comment (tombentley) [inline contact.html]: maling → mailing

File: contact.html [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
 			</li>
 			<li>
-			  <a href="mailto: jira@kafka.apache.org">jira@kafka.apache.org</a>: A list to track Kafka <a href="https://issues.apache.org/jira/projects/KAFKA">JIRA</a> notifications. To subscribe, send an email to <a href="mailto: jira-subscribe@kafka.apache.org">jira-subscribe@kafka.apache.org</a>. Archives available <a href="http://mail-archives.apache.org/mod_mbox/kafka-jira">here</a>.
+				<b>JIRA mailing list</b>: A list to track Kafka&reg; <a href="https://issues.apache.org/jira/projects/KAFKA">JIRA</a> notifications. To subscribe, send an email to <a href="mailto: jira-subscribe@kafka.apache.org">jira-subscribe@kafka.apache.org</a>. Once subscribed, send your emails to <a href="mailto: jira@kafka.apache.org">jira@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-jira">here</a>.
```
Comment (tombentley) [inline contact.html]: "Once subscribed, send your emails to..." I'm not sure that's how the JIRA list is intended to work. It's simply about notification of changes in JIRA; people won't normally mail the list themselves.

File: contact.html (hunk 1 above)
Comment (scholzj) [reply contact.html]: Fixed, thanks for noticing.

File: contact.html (hunk 2 above)
Comment (scholzj) [reply contact.html]: Fixed.

**Review by ewencp — APPROVED** [review]
Seems reasonable to me, thanks for the contribution!

**Pull request merged**